data/schedule.yaml

```diff
@@ -73,17 +74,18 @@ locations:
   - location: Workshop Room
     description: Workshop Track
     events:
-      - name: Google Algorithms Workshop
+      - name: Google - Algorithms Workshop
         by: Eimear Crotty (Google)
         time: '09:30'
-        description: >
+        description: 
                 Planning to attend this workshop?
                 Comeplete the pre-work here -
                 https://sistem.redbrick.dcu.ie/assets/pre_work.pdf
-      - name: SLOT FILLED
+      - name: Microsoft - Building a Modern SPA with Typescript and DotNet.
         by: TBA
         time: '11:30'
-      - name: Problem Solving with Code
+        description: Modern Web Apps are all the rage these days. Through this workshop you’ll get hands-on experience with some powerful technologies powering Microsoft’s SPA’s such as Typescript, ASP.NET Web API, React and Webpack. Combine these to develop a Url Minifier, and, get an insight into the skills needed to become a Full stack developer at Microsoft.
```

Comment:
use multi-line string rather the one line 

note: multiline string are the `>` ones